core/vdbe/datetime.rs

```diff
@@ -661,132 +661,18 @@ mod tests {
 
     #[test]
     fn test_valid_get_time_from_datetime_value() {
-        let now = chrono::Local::now().to_utc().format("%H:%M:%S").to_string();
+        use chrono::{NaiveTime, Offset, Timelike, Utc};
+        use std::{rc::Rc, thread, time::Duration};
+
+        // Setup the datetime value
+        let now = Utc::now().format("%H:%M:%S").to_string();
 
         let test_time_str = "22:30:45";
         let prev_time_str = "20:30:45";
         let next_time_str = "03:30:45";
 
         let test_cases = vec![
-            // Format 1: YYYY-MM-DD (no timezone applicable)
-            (
-                OwnedValue::Text(Rc::new("2024-07-21".to_string())),
-                "00:00:00",
-            ),
-            // Format 2: YYYY-MM-DD HH:MM
-            (
-                OwnedValue::Text(Rc::new("2024-07-21 22:30".to_string())),
```

Comment:
why did you delete so many tests?

Reply:
Sorry, my bad, on the change I have removed by mistake.